core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.ts

```diff
@@ -266,6 +271,40 @@ export class NgbdModalWorkflowExecutionsComponent implements OnInit {
     }
   }
 
+  shortenName(name: string): string {
+    if (name.length <= MAX_TEXT_SIZE) {
+      return name;
+    } else {
+      let words = name.slice(0, MAX_TEXT_SIZE).split(" ");
+      return words.slice(0, -1).join(" ") + "...";
```

Comment:
no need to split at spaces

Reply:
So I need to hardcode the length for the string?